extensions/jms/src/main/java/org/mule/extensions/jms/internal/source/JmsListener.java

```diff
@@ -141,15 +138,15 @@
   @Parameter
   @Optional
   @Example(EXAMPLE_CONTENT_TYPE)
-  private String contentType;
+  private String inputContentType;
 
   /**
-   * The encoding of the message body
+   * The inputEncoding of the message body
    */
   @Parameter
   @Optional
   @Example(EXAMPLE_ENCODING)
-  private String encoding;
+  private String inputEncoding;
```

Comment:
glad to hear new names for this, we need a different name for the content type being read and the content type of the response, but both are living together because of the parameter groups